File: Iniciante/URI 1003.scala
import java.util.Scanner

object Main {

    def main(args: Array[String]) {
        val leitor = new Scanner(System.in)
        val a = leitor.nextInt()
        val b = leitor.nextInt()
        val soma = a + b
        println("SOMA = " + soma)
    }
	
}

File: Iniciante/URI 1003.kt
import java.util.Scanner;

fun main(args: Array<String>) {
    val leitor = Scanner(System.`in`);
    
    val a: Int = leitor.nextInt();
    val b: Int = leitor.nextInt();
    val soma: Int = a + b;
    
    println("SOMA = " + soma);
}